sri4node.ts

```diff
@@ -1551,7 +1451,21 @@ async function configure(app: Application, sriConfig: TSriConfig): Promise<TSriS
 // export = // for typescript
 export {
   configure,
-  debugAnyChannelAllowed as debug, // debugAnyChannelAllowed(ch, msg) => debug(null, ch, msg)
+  /**
+   * @deprecated
+   * This function depends on the configuration object, which is not available before
+   * configure() has been called. Hence it should be removed here, and put in the
+   * sri4nodeServerInstance object that is returned by configure() or passed
+   * as part of every sriRequest, so it can be called whenever needed.
+   */
+  debugAnyChannelAllowed as debug,
```

Comment:
it is exported as debug, so it could probably still be used here and there (but if we release this version as 3.0, we could make some breaking changes, as long as we fix all plugins as well).
We need to talk about this.

Reply:
we removed it (and will hbave to check that no plugin uses it)